.github/workflows/ci_tests.yml

```diff
@@ -155,9 +162,17 @@ jobs:
 
     - name: Run OTE server
       run: |
-        docker-compose -f .github/utils/docker-compose.yml up &
+        docker pull ghcr.io/emmc-asbl/oteapi:${DOCKER_OTEAPI_VERSION}
```

Comment:
Why the change from a separate docker-compose?

Reply:
This is the error, and hence the fix.